homeassistant/components/sensibo/const.py

```diff
@@ -12,3 +17,9 @@
 ALL = ["all"]
 DEFAULT_NAME = "Sensibo"
 TIMEOUT = 8
+
+FIELD_TO_FLAG = {
```

Comment:
We should still keep the translation of sensibo feature to climate feature in the climate platform.